docs/conops.md

```diff
@@ -12,9 +12,70 @@ Maintainer(s) | @arrow-air/services
 
 :exclamation: This document is intended for internal use.
 
+## Table of Contents
+
+<!-- TOC -->
+
```

Comment:
Many headings don't seem to be linking.

Reply:
Agreed with reviewer to remove manual TOC and rely on:
- GitHub Autogenerated TOC: https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/
- Autogenerated Docusaurus TOC

Reply:
https://github.com/Arrow-air/se-services/pull/2/commits/e0f2f98ad6a6085054ba4c248e7a176c73f8ac7c